### DISCUSSION TITLE
Does the FLSUN Q5 support arc commands/Arc Welder?

### DISCUSSION THREAD

**Discussion by jaylw314** [blackfyre/flsun.community#3] [Q&A] [answered]
I'll ask the first question here!

**Comment by blackfyre** [comment, marked as answer]
First of all: Welcome to the Discussions section 😄 
Good job on making the first post 👍 

A quick check can answer this 😄 
Get a GCode console connected to your printer and send `G2`/`G3` through it. If it responds with something like `unknown command` then no, it won't support Arc Welder.
But (yeah, there's always one), you can move over to another firmware that supports it, like: https://github.com/Foxies-CSTL/Marlin_2.0.x/tree/FLSUN_QQS-PRO-MULTI/Marlin/Firmwares
